cmd/pkg-unpack/main.go

```diff
@@ -48,32 +47,35 @@ func Run(ctx context.Context) error {
 		ObjectTyper:   unstructuredscheme.NewUnstructuredObjectTyper(),
 		ObjectCreater: unstructuredscheme.NewUnstructuredCreator(),
 	}
-	p := parser.New(uScheme, uScheme)
+
+	// We want to leverage the package parser, but we don't need to make a
+	// distinction between package metadata and payload objects. Passing en
+	// empty 'metadata' scheme and an unstructured 'object' scheme will cause
+	// the parser to treat any valid Kubernetes YAML (including package metadata
+	// objects) to be treated as regular payload objects. Any object that
```

Comment:
Looks like comment got cut off

Reply:
The dangers of trying to code and meet at the same time. Fixed.